bim2sim/elements/aggregation/bps_aggregations.py

```diff
@@ -138,230 +138,142 @@ def _intensive_calc(self, name) -> ureg.Quantity:
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    ratio_conv_rad_persons = attribute.Attribute(
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    machines = attribute.Attribute(
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    ratio_conv_rad_machines = attribute.Attribute(
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    use_maintained_illuminance = attribute.Attribute(
-        functions=[_bool_calc],
-        dependant_elements='elements'
-    )
-    lighting_power = attribute.Attribute(
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    fixed_lighting_power = attribute.Attribute(
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    ratio_conv_rad_lighting = attribute.Attribute(
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    maintained_illuminance = attribute.Attribute(
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    lighting_efficiency_lumen = attribute.Attribute(
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    use_constant_infiltration = attribute.Attribute(
-        functions=[_bool_calc],
-        dependant_elements='elements'
-    )
-    base_infiltration = attribute.Attribute(
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    max_user_infiltration = attribute.Attribute(
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    max_overheating_infiltration = attribute.Attribute(
-        functions=[_intensive_list_calc],
-        dependant_elements='elements'
-    )
-    max_summer_infiltration = attribute.Attribute(
-        functions=[_intensive_list_calc],
-        dependant_elements='elements'
-    )
-    winter_reduction_infiltration = attribute.Attribute(
-        functions=[_intensive_list_calc],
-        dependant_elements='elements'
-    )
-    min_ahu = attribute.Attribute(
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    max_ahu = attribute.Attribute(
-        functions=[_intensive_calc],
-        dependant_elements='elements'
-    )
-    with_ideal_thresholds = attribute.Attribute(
-        functions=[_bool_calc],
-        dependant_elements='elements'
-    )
-    persons_profile = attribute.Attribute(
-        functions=[_intensive_list_calc],
-        dependant_elements='elements'
-    )
-    machines_profile = attribute.Attribute(
-        functions=[_intensive_list_calc],
-        dependant_elements='elements'
-    )
-    lighting_profile = attribute.Attribute(
-        functions=[_intensive_list_calc],
-        dependant_elements='elements'
-    )
+    t_set_cool = bps.ThermalZone.t_set_cool.to_aggregation(_intensive_calc)
```

Comment:
How do these work and hy do you use them instead of the previous attributes? Do they still have the same functionality as before?

Reply:
the functionality is show in the `to_aggregation` function of the `attribute`class. Its just a new attribute with the same functionality.